services/src/pro/handlers/users.rs

```diff
@@ -1286,7 +1291,11 @@ mod tests {
             ),
         );
 
-        let ctx = ProInMemoryContext::new_with_oidc(request_db, TestDefault::test_default());
+        let ctx = ProInMemoryContext::new_with_oidc(
+            request_db,
+            TestDefault::test_default(),
+            get_config_element::<crate::pro::util::config::Quota>().unwrap(),
+        );
```

Comment:
does this test need to read the config.toml? can't we use a test_default here?
Same for all the other test where you now use get_config_element...